src/containers/NdlaFilm/components/ThemeMovies.tsx

```diff
@@ -54,9 +54,6 @@ export const ThemeMovies = ({ movies, onMoviesUpdated, placeholder, comboboxLabe
     pageSize: 10,
     contextTypes: ["standard"],
     sort: "-relevance",
-    resourceTypes: [
-      "urn:resourcetype:documentary,urn:resourcetype:featureFilm,urn:resourcetype:series,urn:resourcetype:shortFilm",
-    ],
```

Comment:
@gunnarvelle jeg får ingen resultater returnert om ikke denne fjernes .. vet du hvorfor?

Reply:
Den må nesten være med for ellers får du resultat i lista som ikkje er filmer. Men den må være slik:
```
    resourceTypes: [
      "urn:resourcetype:documentary",
      "urn:resourcetype:featureFilm",
      "urn:resourcetype:series",
      "urn:resourcetype:shortFilm",
    ],
```